main.go

```diff
@@ -113,11 +119,13 @@ func doPeriodicMode(es *elasticsearch.Elasticer, d *database.Databaser, client *
 
 	// Accept and handle messages sent out with the index.all and index.templates routing keys
 	client.AddConsumerMulti(
-		messaging.ReindexExchange,
-		"direct",
-		"templeton.periodic",
+		amqpExchangeName,
+		amqpExchangeType,
+		amqpQueueName,
```

Comment:
This change I'm not sure I agree with -- by putting the queue name in properties, it separates the naming by function (periodic vs. incremental) from the difference in bindings on the next line, which is the difference in function. Since these will be separated from other envs/etc. by exchange and vhost anyway, is there still a use case for having the queue name be configurable?